examples/ultrahdr_app.cpp

```diff
@@ -631,16 +633,13 @@ bool UltraHdrAppInput::encode() {
   RET_IF_ERR(uhdr_enc_set_using_multi_channel_gainmap(handle, mUseMultiChannelGainMap))
   RET_IF_ERR(uhdr_enc_set_gainmap_scale_factor(handle, mMapDimensionScaleFactor))
 #ifdef PROFILE_ENABLE
-  const int profileCount = 10;
   Profiler profileEncode;
   profileEncode.timerStart();
-  for (auto i = 0; i < profileCount; i++) {
```

Comment:
Curious why we are removing this logic?

Reply:
the first call to uhdr_decode() decodes the bitstream. Repeated calls will return instantly because the context already contains decoded output. So it wont decode again. The context needs to be reset and used again. This concept of trying decode 10 times and then averaging wont work. So i am collecting data in the first iteration only.